pkgdb2client/cli.py

```diff
@@ -985,8 +985,6 @@ def main():
     pkgdbclient.insecure = arg.insecure
 
     if arg.bzurl != pkgdb2client.BZ_URL:
-        if not arg.bzurl.endswith('xmlrpc.cgi'):
-            arg.bzurl = '%s/xmlrpc.cgi' % arg.bzurl
```

Comment:
ok, thanks for the info, I didn't know python-bugzilla was already taking care of it, but as you say, better safe than sorry :)